tests/test_cases/test_cocotb/test_handle.py

```diff
@@ -123,3 +123,93 @@ def test_real_assign_int(dut):
     log.info("Read back value %d" % got)
     if got != float(val):
         raise TestFailure("Values didn't match!")
+
+
+@cocotb.test()
+async def test_int_vector_32bit_signed(dut):
+    """
+    Test signed integer access to 32-bit vector
+    """
+    N = len(dut.stream_in_data_dword)
+    S_MIN = -2**(N-1)
+    S_MAX = 2**(N-1)-1
+
+    for value in [0, 1, -1, 4, -4, S_MIN, S_MAX]:
+        dut.stream_in_data_dword = value
+        await cocotb.triggers.Timer(10, 'ns')
```

Comment:
```suggestion
        await Timer(10, 'ns')
```
and below; unless you had a specific reason (which then isn't obvious).

Reply:
Thanks for the suggestion, still learning about await.